pages/api/Schema/ContactFilters/typeDefs.ts

```diff
@@ -0,0 +1,34 @@
+import { gql } from 'graphql-tag';
```

Comment:
This file seems like a duplicate of the `.graphql` file. Is it needed? Can codegen make it? Or should we use this instead of the `.graphql` file?

Reply:
So we need ```.graphql``` file for codegen and then the ```typeDefs``` files are what is actually used for the logic to generate the schema. I can try again to use the ```.graphql``` file but I think I had issues with it if I remember correctly.

Reply:
Actually, it does work! My error was probably before I got the wildcards working.